lib/modules/manager/npm/types.ts

```diff
@@ -0,0 +1,75 @@
+interface LockFilePackage {
+  name?: string;
+  version?: string;
+  resolved?: string;
+  integrity?: string;
+  link?: boolean;
+  dev?: boolean;
+  optional?: boolean;
+  devOptional?: boolean;
+  inBundle?: boolean;
+  hasInstallScript?: boolean;
+  hasShrinkwrap?: boolean;
+  bin?: string | { [index: string]: string };
+  license?: string;
+  engines?: { [index: string]: string };
```

Comment:
Please use `Record` type instead of indexer type

Reply:
oh thx, that's way better